src/pages/Login/Login.js

```diff
@@ -1,7 +1,46 @@
 import React, { Component } from 'react';
 import './Login.scss';
+import { SIGNIN_API } from '../../config';
 
 export class Login extends Component {
+  state = {
+    id: '',
+    password: '',
+  };
+
+  login = e => {
+    e.preventDefault();
+    fetch(SIGNIN_API, {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.password,
+      }),
+    })
+      .then(res => res.json())
+      .then(result => {
+        localStorage.setItem('token', result.token);
+      });
+    this.props.history.push('/');
```

Comment:
- 로직 흐름상, 1. 응답을 받아온 후 2. 토큰을 로컬 스토리지에 저장하고 3. 페이지 이동을 하는 순서로 진행이 되어야 할 것 같습니다.
- 따라서 push 함수 실행이 then 구문 안으로 들어가야 합니다!
```suggestion
      .then(result => {
        localStorage.setItem('token', result.token);
        this.props.history.push('/');
      });

```

Reply:
네 수정하겠습니다~~